docs/source/guide/recommender_models.rst

```diff
@@ -0,0 +1,80 @@
+Recommender Models
+=====================
+
+This guide explains the different type of recommender models with a focus on deep-learning based ones.
+
+.. contents:: Table of Contents
+   :local:
+   :backlinks: none
+.. section-numbering::
+
+
+A ``recommender model`` is a single model, e.g. rule-based or trained one, which outputs a score or set of candidates. Generating final recommendations often requires multiple steps, called ``recommender systems``. There is more information on recommender systems in the next guide (will follow soon). 
+
+Ranking Models
+-------------
+
+.. image:: ./imgs/models_ranking.png
+
+**Ranking models** are probably the most common use-case in recommender systems. When the community talks about recommender models, they probably mean ranking models. 
+
+The ranking model predicts a specific target. For example, it could be the review a customer would give an item or it could be the likelihood a user will click on an item. Click-through-rate prediction is a typical use-case for that. The Figure 1a above provides a simple example of a dataset. The dataset has different input features (``user_id``, ``user_age``, etc.) and the target (``click``). The dataset contains both the positive (``click=1``) and negative (``click=0``) interaction data. If negative examples are not available, then negative sampling strategies are required. An example architecture is visualized by Figure 1b. The inputs are concatenated and fed through an Multi-Layer Perceptron (MLP) Block (multiple fully connected layers).
+
+Merlin Models provides implementation of popular architectures, such as MLP, NCF, Facebook’s DLRM, Google’s DCN and Google’s Wide&Deep networks with TensorFlow backend. Classical machine learning models, such as implicit or XGBoost can be used in Merlin Models, as well.
+
+**Resources**:
```

Comment:
Consider making this an H3.  Maybe also make it "Resources for Ranking Models." Based on the [sections](https://docutils.sourceforge.io/docs/user/rst/quickstart.html#sections) info from docutils, underlining with `~` would do it.  If you set `:depth: 2` in the `contents` directive at the top, it won't bugger your TOC.  If you really don't want to make it a heading, consider making it a [rubric](https://docutils.sourceforge.io/docs/ref/rst/directives.html#rubric): `.. rubric:: Resources`.  (I can't explain why those two bits of info aren't on the same docutils HTML page.)

For the list of links, my sugg is to adopt intersphinx. The primary benefit is that you'd find out if a link breaks during a build rather than a reader bothering to open an issue or PR.  There's a little bit of lifting involved. We'd need to update the `conf.py` for this project with a mapping like I did for [nvt](https://github.com/NVIDIA-Merlin/NVTabular/blob/b1fad2a4e8f3bee08d8e795948757f9d1d0e8f30/docs/source/conf.py#L134-L141).  It's prolly also time to update Sphinx to `<=4.6`.  After all that, then markup like the foll creates the link *and makes sure the links is good at build time*:

```
* :external+merlin-models:std:doc:`examples/03-Exploring-different-models`
```

If any of that is appealing, I can see if I can make the changes I suggested one of these evenings or maybe next weekend.